packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -49,17 +49,28 @@ const MyAdsTable = () => {
             <React.Fragment>
                 {selected_advert && <QuickAddModal advert={selected_advert} />}
                 {my_ads_store.has_missing_payment_methods && (
-                    <div className='p2p-my-ads__warning'>
-                        <HintBox
-                            icon='IcAlertWarning'
-                            message={
-                                <Text as='p' size='xxxs' color='prominent' line_height='xs'>
-                                    <Localize i18n_default_text="Some of your ads don't contain payment methods. To make it easier for people to pay you, please add payment methods to all your ads." />
-                                </Text>
-                            }
-                            is_warn
-                        />
-                    </div>
+                    <HintBox
+                        className='p2p-my-ads__warning'
+                        icon='IcAlertWarning'
+                        message={
+                            <Text as='p' size='xxxs' color='prominent' line_height='xs'>
+                                <Localize i18n_default_text="Some of your ads don't contain payment methods. To make it easier for people to pay you, please add payment methods to all your ads." />
+                            </Text>
+                        }
+                        is_warn
+                    />
+                )}
```

Comment:
My suggestion is to make this to a component and split it to its own file. Copy pasting like this makes it hard to track when you want to remove it later. If you make it to a component and to its own file, when you delete the file, ESLint will throw errors and now you can track where you should remove the temporary components.

Reply:
ok.. Will move it to a new file with comments